examples/generative/corrdiff/conf/config_generate.yaml

```diff
@@ -18,7 +18,7 @@ hydra:
 
 
 ## Main options
```

Comment:
Some of the default configs are changed here, including the batch size and validation times. Is that necessary?